src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,228 @@
+                ui.listSizeMessage(taskList);
+                break;
+            case DEADLINE:
+                String deadlineDescription = ui.readCommandLine();
+                if (!deadlineDescription.contains(" /by ")) {
+                    try {
+                        ui.setIndentedLine();
+                        throw new DukeException("Sorry, " +
+                                "please give me a description of the deadline as well! >.<\n" +
+                                "  " +
+                                "Format should be deadline (description) /by (yyyy-MM-dd HHmm!)");
+                    } catch (DukeException e) {
+                        System.out.println("  " + e.getMessage());
+                        ui.setIndentedLine();
+                        return;
+                    }
+                }
+                String[] deadlineArguments = deadlineDescription.split(" /by ");
+                String DLDescription = deadlineArguments[0];
+                String dateTime = deadlineArguments[1];
+
+                try {
+                    Deadline deadline = new Deadline(DLDescription, dateTime);
+                    taskList.addTask(deadline);
+                    ui.setIndentedLine();
+                    System.out.println("  " + deadline);
+                    ui.listSizeMessage(taskList);
+                    break;
+                } catch (DateTimeParseException e) {
+                    System.out.println("Sorry! " +
+                            "Format should be deadline (description) /by (yyyy-MM-dd HHmm!)");
+                    ui.setIndentedLine();
+                    return;
+                }
+            case EVENT:
+                String eventDescription = ui.readCommandLine();
+                if (!eventDescription.contains(" /from ") || !eventDescription.contains(" /to ")) {
+                    try {
+                        ui.setIndentedLine();
+                        throw new DukeException("Sorry, " +
+                                "please give me a description of the event as well! >.<\n" +
+                                "  " +
+                                "Format should be event " +
+                                "(description) /from (yyyy-MM-dd HHmm) /to (yyyy-MM-dd HHmm)!");
+                    } catch (DukeException e) {
+                        System.out.println("  " + e.getMessage());
+                        ui.setIndentedLine();
+                        return;
+                    }
+                }
+                String[] eventArguments = eventDescription.split(" /from ");
+                String[] eventDuration = eventArguments[1].split(" /to ");
+                String EVDescription = eventArguments[0];
+                String startTime = eventDuration[0];
+                String endTime = eventDuration[1];
+
+                try {
+                    Event event = new Event(EVDescription, startTime, endTime);
```

Comment:
is EV in EVDescription an abbreviation?
if so i think it should be Ev to fit the style guide

Reply:
Yep it was supposed to mean EventDescription. Thx for spotting that out, will change it!